templates/org/times/daterange.tmpl

```diff
@@ -0,0 +1,15 @@
+<div class="ui form">
+	<div class="field">
+		<label>{{$.i18n.Tr "Start date"}}</label>
```

Comment:
Please use the key specified in the `options/locale/locale_en-US.ini` file.

Reply:
Changed.